qiskit_experiments/analysis/curve_analysis.py

```diff
@@ -669,14 +669,79 @@ def _dictionarize(parameter_name):
         return fitter_options
 
     @property
-    def _num_qubits(self):
+    def _experiment_type(self) -> str:
+        """Return type of experiment."""
+        try:
+            return self.__experiment_metadata["experiment_type"]
+        except (TypeError, KeyError):
+            # Ignore experiment metadata is not set or key is not found
+            pass
+
+    @property
+    def _num_qubits(self) -> int:
         """Getter for qubit number."""
-        return len(self.__qubits)
+        try:
+            return self.__experiment_metadata["num_qubits"]
+        except (TypeError, KeyError):
+            # Ignore experiment metadata is not set or key is not found
+            pass
 
     @property
-    def _physical_qubits(self):
+    def _physical_qubits(self) -> List[int]:
         """Getter for physical qubit indices."""
-        return list(self.__qubits)
+        try:
+            return list(self.__experiment_metadata["physical_qubits"])
+        except (TypeError, KeyError):
+            # Ignore experiment metadata is not set or key is not found
+            pass
+
+    def _experiment_options(self, index: int = -1) -> Dict[str, Any]:
+        """Return the latest experiment options.
```

Comment:
Since whether it's the latest job depends on `index`, rephrase to something like "Return experiment options of a given job". Same for the other propoerties.

Reply:
Good catch! Fixed in https://github.com/Qiskit/qiskit-experiments/pull/150/commits/09e285ba056df32eddc26ad07cbfd92f6470f7e7 Thank you!